conductr_cli/bndl_main.py

```diff
@@ -257,6 +296,21 @@ def add_conf_arguments(parser):
                                     default=[],
                                     action=StartCommandAction)
 
+    volume_args = parser.add_argument_group('volumes',
+                                            description='Declare volumes for a component.'
+                                                        'If the bundle has more than one component, you must specify '
+                                                        '--component\n'
+                                                        'Example: bndl --vol')
+
+    volume_args.add_argument('--volume',
+                             dest='volume_dicts',
+                             action=VolumeAction,
+                             default=[],
+                             help='Declare a volume path given its name and mount point separated by a colon\n'
+                                  'If provided, default volumes are removed\n'
+                                  'Example: bndl --volume my-vol:/data --component web',
+                             metavar='NAME:MOUNT-POINT')
```

Comment:
Feedback incorporated, thank you. I agree that being consistent with our `KEY=VALUE` convention is better here, so this was changed to `VOL=path` e.g. `my-vol=/data`.